src/commands/settings/automod.ts

```diff
@@ -689,30 +691,29 @@ const mentionMenu = async (
                         : "") +
                     "\n" +
                     `Except if...\n` +
-                    `> ${
-                        current.allowed.users.length > 0
-                            ? `Member is: ${listToAndMore(
-                                  current.allowed.users.map((u) => `<@${u}>`),
-                                  3
-                              )}\n`
-                            : ""
-                    }` +
-                    `> ${
-                        current.allowed.roles.length > 0
-                            ? `Member has role: ${listToAndMore(
-                                  current.allowed.roles.map((r) => `<@&${r}>`),
-                                  3
-                              )}\n`
-                            : ""
-                    }` +
-                    `> ${
-                        current.allowed.channels.length > 0
-                            ? `In channel: ${listToAndMore(
-                                  current.allowed.channels.map((c) => `<#${c}>`),
-                                  3
-                              )}\n`
-                            : ""
-                    }`
+                    (current.allowed.users.length > 0
+                        ? `> Member is: ${listToAndMore(
+                              current.allowed.users.map((u) => `<@${u}>`),
+                              3
+                          )}\n`
+                        : "") +
+                    (current.allowed.roles.length > 0
+                        ? `> Member has role: ${listToAndMore(
+                              current.allowed.roles.map((r) => `<@&${r}>`),
+                              3
+                          )}\n`
+                        : "") +
+                    (current.allowed.channels.length > 0
+                        ? `> In channel: ${listToAndMore(
+                              current.allowed.channels.map((c) => `<#${c}>`),
+                              3
+                          )}\n`
+                        : "") +
+                    (current.allowed.users.length == 0 ||
+                    current.allowed.roles.length == 0 ||
+                    current.allowed.channels.length == 0
+                        ? "> *No exceptions*\n"
```

Comment:
If we're renaming exceptions to "allowed list" should this text also be changed?